README.md

```diff
@@ -35,8 +36,19 @@ npm install fiori-fundamentals --save
 
 > NOTE: We only distribute the [fonts, icons and CSS](https://github.com/SAP/fundamental/tree/master/dist) and [the SASS source](https://github.com/SAP/fundamental/tree/master/scss), not the full project or HTML for specific components.
 
+
+### LESS compatible Fiori Fundamentals
+
+The distributed `less` folder is NOT a full port of the library to LESS. We are providing only select files that can be used in a LESS environment. For example, some `calc` functions are escaped to avoid errors when the library is processed as LESS.
+
+> NOTE: The `fonts` and `icons` are not included as part of this file. You will need to bring the regular `dist` versions of those files into your project separately.
+
+We are offering these files to ease the transition for some teams that are migrating their UI to a new look and feel but who are not in a position to adopt a SASS workflow.
+
+We welcome ideas and contributions to build this out if there is a strong need.
+
+
```

Comment:
is LESS being generated with every build?

Reply:
prod builds yes.